frontend/src/routes/[name]/+page.svelte

```diff
@@ -0,0 +1 @@
+<h1>This page will display user profile</h1>
```

Comment:
the logical address for this page is 'me' and not 'name'

Reply:
In devise we have "current_user.name" not "current_user.me"

Reply:
And here "current_user.name" to the address bar?
@npupko what do you say?

Reply:
```
<script>
 let name = 
  background.get(current_user.name) #ПСЕВДОКОД
</script>
```
href="/{$name}"

Reply:
Same as in comment above. Let's move this to the correct url